### ISSUE TITLE
Mouse over enemies/resources

### ISSUE THREAD

**Issue by dangranos** [gloomyson/StarCraft#27]
"Moving" at enemies uses same pointer animation as normal moving, also does not makes units attack enemies.

**Comment by gloomyson** [comment]
No, moving at enemy will make unit attack enemy, you should check it again.

**Comment by dangranos** [comment]
Oh, maybe i didn't noticed that. Though there is still the graphical part of the issue left.
